copy/copy.go

```diff
@@ -180,6 +180,10 @@ type Options struct {
 	// Download layer contents with "nondistributable" media types ("foreign" layers) and translate the layer media type
 	// to not indicate "nondistributable".
 	DownloadForeignLayers bool
+
+	// Give priority to pulling zstd images if multiple images are present unless explicitly
+	// configured true.
+	PreferGzip types.OptionalBool
```

Comment:
- Conceptually this belongs in the image selection block, maybe after `Instances`
- The documentation must say that this affects only `CopySystemImage`
- … and, I think, that we
  - prefer gzip (not zstd, as the comment says now) images if this is set to `OptionalBoolTrue`
  - prefer the best compression supported by the system (which may change over time) if this set to `OptionalBoolFalse`
  - choose automatically (and the choice may change over time) if this is set to `OptionalBoolUndefined` (which is the default behavior, and recommended for most callers).

Cc: @vrothberg